ts/ccxt.ts

```diff
@@ -77,7 +77,6 @@ import bitstamp from  './src/bitstamp.js'
 import bitstamp1 from  './src/bitstamp1.js'
 import bittrex from  './src/bittrex.js'
 import bitvavo from  './src/bitvavo.js'
-import bkex from  './src/bkex.js'
```

Comment:
@samgermain next time, you don't need to remove it from here, this part of the file is automatically generated

Reply:
ok, but the whole file isn't automatically generated? Because I believe the build was failing before removing stuff from this file